sklearn/feature_selection/tests/test_base.py

```diff
@@ -1,12 +1,15 @@
 import numpy as np
 from scipy import sparse as sp
+from scipy.stats import pearsonr, spearmanr
```

Comment:
Have a look at my latest commit. I removed `**kwargs` from `call_scorer` and put it in `featurewise_scorer` because only there the user will have the freedom to put the kwargs.